phys2bids/tests/test_txt.py

```diff
@@ -1,185 +1,127 @@
-import os
-from pkg_resources import resource_filename
-from phys2bids.interfaces import txt
-from pytest import raises
-import wget
+import math
+
 import pytest
+from pytest import raises
+
+from phys2bids.interfaces import txt
 
 
-@pytest.fixture
-def samefreq_acq_file():
-    url = 'https://osf.io/4yudk/download'  # url to Test_belt_pulse_samefreq.txt
-    test_path = resource_filename('phys2bids', 'tests/data')
-    test_filename = 'Test_belt_pulse_samefreq_short.txt'
-    test_full_path1 = os.path.join(test_path, test_filename)
-    if not os.path.isfile(test_full_path1):
-        wget.download(url, test_full_path1)
-    # chtrig = 2
-    return test_full_path1
-
-
-@pytest.fixture
-def multi_lab_file():
-    url = 'https://osf.io/q4x2f/download'
-    # url to Test_2minRest_trig_multifreq_header_comment.txt
-    test_filename = 'Test_2minRest_trig_multifreq_header_comment.txt'
-    test_path = resource_filename('phys2bids', 'tests/data')
-    test_full_path2 = os.path.join(test_path, test_filename)
-    if not os.path.isfile(test_full_path2):
-        wget.download(url, test_full_path2)
-    # chtrig = 1
-    return test_full_path2
-
-
-@pytest.fixture
-def no_time_file():
-    # use file without time column
-    # # url to the file Test2_trigger_CO2_O2_pulse_1000Hz_534TRs_no_time.txt
-    url = 'https://osf.io/u5dq8/download'
-    test_filename = 'Test2_trigger_CO2_O2_pulse_1000Hz_534TRs_no_time.txt'
-    test_path = resource_filename('phys2bids', 'tests/data')
-    test_full_path3 = os.path.join(test_path, test_filename)
-    if not os.path.isfile(test_full_path3):
-        wget.download(url, test_full_path3)
-    # chtrig = 0
-    return test_full_path3
-# header_T2ntime, channels_T2ntime = txt.read_header_and_channels(test_full_path3, chtrig)
-
-
-@pytest.fixture
-def test_read_header_and_channels(samefreq_acq_file, multi_lab_file):
-    header_acq, channels_acq = txt.read_header_and_channels(samefreq_acq_file, 2)
-    header_lab, channels_lab = txt.read_header_and_channels(multi_lab_file, 1)
+@pytest.fixture(scope='function')
+def loaded_acq_file(samefreq_short_txt_file):
+    chtrig = 2
+    header_acq, channels_acq = txt.read_header_and_channels(samefreq_short_txt_file, chtrig)
+
+    # just a few quick checks to make sure the data loaded correctly
     assert len(header_acq) == 8  # check proper header lenght
     assert len(channels_acq) == 1048560  # check proper number of timepoints
     assert len(header_acq[-1]) == 2  # check extra line is deleted
-    # load file with comment
-    # url to Test_2minRest_trig_multifreq_header_comment.txt
-    assert len(channels_lab[152109 - 9]) == 6  # check the comment has been eliminated
-    return header_acq, channels_acq, header_lab, channels_lab
+    assert 'acq' in header_acq[0][0]
 
+    return header_acq, channels_acq, chtrig
 
```

Comment:
Hi Ross, the ``Test_2minRest_trig_multifreq_header_comment.txt`` was been used here because it has a comment iniside that the program has to eliminate with an if condition. Probably this is activated somewhere else now that you have refractored ```test_txt.py`` but are we sure?

Reply:
I checked codecov and the line is tested so don't worry